text-ui-test/EXPECTED.TXT

```diff
@@ -20,6 +20,25 @@ Choose your starting color to start new game! [b/w] Invalid input! Please enter
 Great! Starting new game as WHITE
 _________________________________________________________________
 
```

Comment:
No need to test for IO redirection moving forward since it is removed from CI checks! :)